server/manager/src/main/java/org/apache/accumulo/master/tableOps/Utils.java

```diff
@@ -198,19 +197,21 @@ public static void checkNamespaceDoesNotExist(ServerContext context, String name
    * Given an input stream and a flag indicating if the file info is base64 encoded or not, retrieve
    * the data from a file on the file system. It is assumed that the file is textual and not binary
    * data.
+   *
```

Comment:
Looks like the comments are outdated now that you changed the parameters.

Reply:
Oh, probably. I can check.

Reply:
Actually, I think this javadoc is still okay.